modules/ROOT/pages/_partials/release_notes/sync-gateway-3-1-5-release-note.adoc

```diff
@@ -0,0 +1,64 @@
+== 3.1.5 -- April 2024
+
+Version 3.1.5 of Sync Gateway delivers the following features and enhancements:
+
+NOTE: For an overview of the latest features offered in Sync Gateway 3.1, see xref:whatsnew.adoc[New in 3.1].
+
+[WARNING]
+--
+
+We have discovered a critical issue impacting a few customers that utilize OIDC. 
+This issue could lead to users losing access to channels, consequently resulting in document revocations.
+We suggest all users revert their Sync Gateway to version 3.1.3 as a precautionary step or promptly upgrade to version 3.1.5.
```

Comment:
Now `3.1.5` is available there's no reason for anyone to downgrade. I think we should only suggest upgrading to `3.1.5`

Reply:
Okay will change!